space-plugins/nuxt-base/app.config.ts

```diff
@@ -14,6 +14,7 @@ declare module '@nuxt/schema' {
 			initOauthFlowUrl: string;
 			successCallback: string;
 			errorCallback: string;
+			disabledMiddlewarePaths?: string[]; //e.g.: ['/api/endpoint']
```

Comment:
Suggestions for naming

Reply:
``` 
disableMiddlewarePaths?: {
    server: string[]
}
```

Reply:
I think this gives an impression that it will disable all the middlewares, whereas we just want to bypass auth middleware. 

- authSkippedPaths
- publicPaths
- unprotectedPaths
- ... still thinking 🤔

Reply:
Considering the fact that this config is within `auth` object, it could be simpler...

```js
{
  "auth": {
    ...
    pathsToSkip: [], // bypassedPaths, ... ummm again not so different actually 😂
  }
}
```

Reply:
what about

```
middleware: {
    server: {
        ignorePaths: []
    }
}
```

Reply:
I think having the word `server` is necessary as we are only enabling this logic on server even though there are middlewares on client side as well. 
```
middleware: {
    ignoreServerPaths: []
}

```

Reply:
@BibiSebi but if you add something like "/info" there, and create a `pages/info.vue`, it should work. It works for both client and server side navigation, right?

Reply:
Correct, so it should only target api endpoints, right? 

``` 
{ auth: { . . middleware: { ignoreAPIPaths: [] } } }
```
or like this? with a comment saying that these should paths starting with api ? 
``` 
{ auth: { . . middleware: { ignorePaths: [] } } }
```